packages/synapse-react-client/src/components/RssFeedCards/RssFeedCards.tsx

```diff
@@ -15,9 +14,14 @@ type RssState = {
   allItemsUrl?: string
 }
 
+export type RssFilter = {
+  type?: 'tag' | 'category'
+  value: string
+}
+
 export type RssFeedCardsProps = {
   url: string
-  lockedColumn?: LockedColumn // optional tag to filter by, typically set by using this component on a DetailsPage
```

Comment:
Unclear why it was using the LockedColumn type, since it didn't really make sense here.  The new type has the option to set the filter type to either 'tag' (which is default) or 'category'.  This determines the WordPress feed URL to use